src/number_theory/bernoulli_polynomials.lean

```diff
@@ -28,8 +28,8 @@ Bernoulli polynomials are defined using `bernoulli`, the Bernoulli numbers.
 ## Main theorems
 
 - `sum_bernoulli`: The sum of the $k^\mathrm{th}$ Bernoulli polynomial with binomial
-  coefficients up to n is `(n + 1) * X^n`.
-- `bernoulli_generating_function`: The Bernoulli polynomials act as generating functions
+  coefficients up to `n` is `(n + 1) * X^n`.
+- `polynomial.bernoulli_generating_function`: The Bernoulli polynomials act as generating functions
```

Comment:
Should the previous list item get a namespace added too?

Reply:
That one isn't namespaced, so the link already works, do you mean should it be in a namespace?